src/accelerate/utils/operations.py

```diff
@@ -557,25 +557,29 @@ def reduce(tensor, reduction="mean"):
             The data to reduce.
         reduction (`str`, *optional*, defaults to `"mean"`):
             A reduction method. Can be of "mean", "sum", or "none"
+        scale (`float`, *optional*):
+            A default scaling value to be applied after the reduce, only valied on XLA.
 
     Returns:
         The same data structure as `data` with all the tensors reduced.
     """
 
-    def _reduce_across_processes(tensor, reduction="mean"):
+    def _reduce_across_processes(tensor, reduction="mean", scale=1.0):
         state = PartialState()
         cloned_tensor = tensor.clone()
         if state.distributed_type == DistributedType.NO:
             return cloned_tensor
         if state.distributed_type == DistributedType.TPU:
-            xm.all_reduce("sum", cloned_tensor)
+            xm.all_reduce("sum", cloned_tensor, scale)
```

Comment:
I'm not sure if this implementation is appropriate, i.e., using the scale parameter only on XLA and ignoring it in other scenarios.

Reply:
In this case it is, as `scale` is only an arg on xla :)

Reply:
Oh, one thing I notice now: This would not work for `reduction="mean"` because it would apply scaling twice, right? Should we raise an error here in that case, or, below, inside of `if reduction == "mean":`, place a guard for `DistributedType.TPU`?

Reply:
Nice catch! Sorry for not noticing this in time, I will submit a PR to add the necessary warnings soon